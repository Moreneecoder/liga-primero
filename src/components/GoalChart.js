import { Link } from 'react-router-dom';
import '../stylesheets/GoalChart.css';
import footballer from '../images/footballer.svg';

const GoalChart = () => (
  <div className="GoalChart px-1 px-md-0 col-12 col-md-6 mx-auto text-center">
    <img className="mb-3" style={{ width: '50%' }} src={footballer} alt="not-found" />
    <div className="alert alert-info p-2">
      <span className="fw-bolder">Goal Chart</span>
      {' '}
      Feature Coming Soon! In the Pipes...
    </div>
    <Link to="/" className="text-white btn btn-md btn-md-lg main-bg-color">&#8592; Go Back To standings</Link>
  </div>
);

export default GoalChart;
