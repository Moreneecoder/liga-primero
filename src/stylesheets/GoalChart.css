.GoalChart {
  padding-top: 30%;
}

@media only screen and (min-width: 768px) {
  .GoalChart {
    padding-top: 15%;
  }
}
