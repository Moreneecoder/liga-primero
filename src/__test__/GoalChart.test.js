import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import GoalChart from '../components/GoalChart';

describe('GoalChart Component', () => {
  test('Renders GoalChart component as expected', () => {
    const tree = renderer.create(<BrowserRouter>
      <GoalChart />
    </BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
