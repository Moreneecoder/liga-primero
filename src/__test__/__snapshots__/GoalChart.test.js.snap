// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoalChart Component Renders GoalChart component as expected 1`] = `
<div
  className="GoalChart px-1 px-md-0 col-12 col-md-6 mx-auto text-center"
>
  <img
    alt="not-found"
    className="mb-3"
    src="footballer.svg"
    style={
      Object {
        "width": "50%",
      }
    }
  />
  <h3
    className="alert alert-info p-2"
  >
    <span
      className="fw-bolder"
    >
      Goal Chart
    </span>
     
    Feature Coming Soon! In the Pipes...
  </h3>
  <a
    className="text-white btn btn-md btn-md-lg main-bg-color"
    href="/"
    onClick={[Function]}
  >
    ‚Üê Go Back To standings
  </a>
</div>
`;
